lib/datum/railtie.rb

```diff
@@ -1,12 +1,11 @@
-module Datum
 require 'rails'
 
-# @!visibility private
-class Railtie < Rails::Railtie
+module Datum
```

Comment:
Won't the `require 'rails'` statement fail when datum is loaded in an environment without Rails installed?

Instead of requiring rails, shouldn't it check if Rails is defined before attempting to define the `Railtie`?

Maybe wrap the require `datum/railtie` call in `datum.rb` with a `if defined?(Rails)` statement? Like pry does?

Reply:
So in a clean Ruby environment (no gems installed) this simple script:
```
require 'datum'

Datum::Datum.new(:first, :last, :middle)
```

Fails with this:
```
/Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- rails (LoadError)
	from /Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from script.rb:1:in `<main>'
Erics-MacBook-Pro:dummy eperkins$ nano script.rb
Erics-MacBook-Pro:dummy eperkins$ ruby script.rb
/Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require': cannot load such file -- rails (LoadError)
	from /Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/gems/2.3.0/gems/datum-4.2.0/lib/datum/railtie.rb:1:in `<top (required)>'
	from /Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:55:in `require'
	from /Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/gems/2.3.0/gems/datum-4.2.0/lib/datum.rb:7:in `<top (required)>'
	from /Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:127:in `require'
	from /Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:127:in `rescue in require'
	from /Users/eperkins/.rbenv/versions/2.3.2/lib/ruby/2.3.0/rubygems/core_ext/kernel_require.rb:40:in `require'
	from script.rb:1:in `<main>'
```